chao_garden/src/agb_flash_mx_512k.c

```diff
@@ -2,17 +2,13 @@
 #include "gba/flash_internal.h"
 
 const u16 mx512kMaxTime[] = {
-    10,   65469, TIMER_ENABLE | TIMER_INTR_ENABLE | TIMER_256CLK,
-    10,   65469, TIMER_ENABLE | TIMER_INTR_ENABLE | TIMER_256CLK,
-    2000, 65469, TIMER_ENABLE | TIMER_INTR_ENABLE | TIMER_256CLK,
-    2000, 65469, TIMER_ENABLE | TIMER_INTR_ENABLE | TIMER_256CLK,
+    10,   65469, TIMER_ENABLE | TIMER_INTR_ENABLE | TIMER_256CLK, 10,   65469, TIMER_ENABLE | TIMER_INTR_ENABLE | TIMER_256CLK,
```

Comment:
This seems worse? Probably needs some comments to keep it the same shape as before